.concourse/pipeline.yaml.gomplate

```diff
@@ -419,11 +421,11 @@ jobs:
 {{- end }}
   - put: s3.kubecf-ci
     params:
-      file: output/kubecf-v*.tgz
+      file: output/kubecf-[0-9]*.tgz
```

Comment:
Shouldn't this contain at least `.`? I think we'd get git commit IDs and things like that, too (for non-release builds).

Reply:
I don't think so. Note that this is not a regexp, but shellscript expansion: first a number, then whatever.

This change is motivated by being able to select the file `kubecf-0.0.0-f541d0f.tgz` but not select `kubecf-bundle-0.0.0-f541d0f.tgz`. Since we don't have the `v0.0.0` anymore, nor lookahead in shellscript expansion, we need something like this.